pom.xml

```diff
@@ -231,6 +231,12 @@
       <version>${guice.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <version>1</version>
```

Comment:
Property for version

Reply:
Always going to be 1 because its built-in to the jdk